### PR TITLE
ashimite lore made consistent and fixed grammar issue

### PR DESCRIPTION
**Pull request by c-sig**
a

### CODE REVIEW

**Review by 0ffz — APPROVED** [review]

File: servers/minecraft/plugins/Mobzy/mobs/passive/ashimite.yml [hunk 1]
```diff
@@ -18,17 +18,18 @@ staticComponents:
           custom-model-data: 1
           display-name: §x§2§5§9§7§D§9Raw Ashimite Meat
           lore: |
-            §eCan be obtained by hunting down
-            §ean §3Ashimite §ein the §aSea of Corpses§e.
+            §eCan be obtained by hunting down an
+            §x§2§5§9§7§D§9Ashimite §ein the §aGoblets of Giants
+            §eand the §aSea of Corpses§e.
```
Comment (Boy0000) [inline servers/minecraft/plugins/Mobzy/mobs/passive/ashimite.yml:23]: Ashimites only spawn in L5 as of right now

File: servers/minecraft/plugins/Mobzy/mobs/passive/ashimite.yml [hunk 2]
```diff
@@ -18,17 +18,18 @@ staticComponents:
...
         cooked:
           type: COOKED_PORKCHOP
           display-name: §x§2§5§9§7§D§9Cooked Ashimite Meat
           custom-model-data: 1
           lore: |
             §eCan be obtained by hunting down an
-            §x§2§5§9§7§D§9Ashimite §ein the §aSea of Corpses
-            §eand cooking its meat.
+            §x§2§5§9§7§D§9Ashimite §ein the §aGoblets of Giants
+            §eand the §aSea of Corpses§eand cooking its meat.
```
Comment (Boy0000) [inline servers/minecraft/plugins/Mobzy/mobs/passive/ashimite.yml:32]: again, only l5

File: servers/minecraft/plugins/Mobzy/mobs/passive/ashimite.yml [hunk 3]
```diff
@@ -53,9 +54,9 @@ staticComponents:
             §x§2§5§9§7§D§9Ashimite §ein the §aGoblets of Giants
             §eand the §aSea of Corpses§e.
             §x§8§2§4§E§2§1§oThey use these to assert dominance.
-            §x§8§2§4§E§2§1§oYet if you get close they run of...
+            §x§8§2§4§E§2§1§oYet they run off when you get close...
```
Comment (Boy0000) [inline servers/minecraft/plugins/Mobzy/mobs/passive/ashimite.yml:57]: sure this is fine

**Review by Boy0000 — CHANGES_REQUESTED** [review]

File: servers/minecraft/plugins/Mobzy/mobs/passive/ashimite.yml (hunk 3 above)
Comment (c-sig) [reply servers/minecraft/plugins/Mobzy/mobs/passive/ashimite.yml:54]: should be fixed now i think?

**Review by Boy0000 — APPROVED** [review]
seems fine iguess

**Pull request merged**